source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/paths.ts

```diff
@@ -0,0 +1,13 @@
+// Top level routes
+export const rootPath = '/';
```

Comment:
I think the next iteration of this would look something like this:

```ts
export const routeFor(routeSegment) {
  const { name, path } = routeSegment;

  if (name) {
    return name;
  }

  if (typeof path === 'function') {
    throw new Error('path is a function and expected a name property to be present');
  }

  return path;
}

export const paths = {
  root: {
    path: '/',
  },
  login: { path: '/login', },
  projects: {
    path: '/projects',
    new: { path: /projects/new', },
    show: { 
      name: '/projects/:id',
      path(id: string) { return `/projects/${id}`; },
    },
    edit: { 
      name: '/projects/:id/edit',
      path(id: string) { return `/projects/${id}/edit`; } 
    },
  }
}
```

or something like that. idk. #ReactRouterProblems